awx/main/models/inventory.py

```diff
@@ -1801,3 +1816,891 @@ class Meta:
 
     def get_absolute_url(self, request=None):
         return reverse('api:inventory_script_detail', kwargs={'pk': self.pk}, request=request)
+
+
+# TODO: move to awx/main/models/inventory/injectors.py
+class PluginFileInjector(object):
+    # if plugin_name is not given, no inventory plugin functionality exists
+    plugin_name = None  # Ansible core name used to reference plugin
+    # if initial_version is None, but we have pugin name, injection logic exists,
```

Comment:
`pugin`

![Pugs-up-close-and-personal](https://user-images.githubusercontent.com/214912/54540074-b47f1100-496d-11e9-8d5f-be793007487b.jpg)